include/nncase/targets/neutral_target.h

```diff
@@ -27,7 +27,7 @@ class NNCASE_API neutral_target : public target
     void register_target_independent_passes(const module_type_t &type, ir::transforms::pass_manager &pass_mgr) override;
     void register_target_dependent_passes(const module_type_t &type, ir::transforms::pass_manager &pass_mgr, bool use_ptq) override;
     void register_quantize_annotation_passes(const module_type_t &type, ir::transforms::pass_manager &pass_mgr) override;
-    void register_quantize_passes(const module_type_t &type, ir::transforms::pass_manager &pass_mgr, datatype_t quant_type, datatype_t w_quant_type, bool use_mse_quant_w) override;
+    void register_quantize_passes(const module_type_t &type, ir::transforms::pass_manager &pass_mgr, datatype_t quant_type, std::string_view w_quant_type, bool use_mse_quant_w) override;
```

Comment:
It is something odd that quant_type is datatype_t, while w_quant_type is std::string_view

Reply:
Only w_quant_type can be int4 or int6 and adding int4 to datatype_t is a wrong design